packages/view-types/spatial/src/Spatial.js

```diff
@@ -172,7 +172,10 @@ class Spatial extends AbstractSpatialOrScatterplot {
       geneExpressionColormap,
     } = this.props;
     const getPolygon = hasExplicitPolygons
-      ? (object, { index, data }) => data.src.obsSegmentations.data[index]
+      // Bug introduced from DeckGL v8.6.x to v8.8.x:
+      // Polygon vertices cannot be passed via Uint32Arrays, which is how they might load via Zarr.
+      // For now, a workaround is to cast each vertex to a plain Array.
+      ? (object, { index, data }) => data.src.obsSegmentations.data[index].map(d => Array.from(d))
```

Comment:
I guess the `data` i.e `this.obsSegmentationsData` doesn't change so the output of this is cached - is this right?  Just worried about calling this repeatedly.